app/src/main/res/layout/lessons_chapter_view.xml

```diff
@@ -58,7 +59,7 @@
       android:fontFamily="sans-serif"
       android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? View.IMPORTANT_FOR_ACCESSIBILITY_YES : View.IMPORTANT_FOR_ACCESSIBILITY_NO}"
       android:text="@{viewModel.chapterName}"
-      android:textColor="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @color/oppia_primary_text : @color/oppia_primary_text_30}"
+      android:textColor="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @color/primary_text_color : @color/disabled_text_color}"
```

Comment:
Layout xmls should not use colors from color_palette.xml directly and should be a new color from component_colors.xml. Ditto at other places. (Styles and themes files might be an exception in some cases.)

Reply:
Fixed that